helm/kubeblocks/falkordb-cluster-omnistrate/tests/integration/cluster/test_cluster_integration.py

```
@@ -0,0 +1,1936 @@
+        # In single-zone test environments (like kind), we verify proper distribution intent
+        if len(unique_zones) == 1:
+            logger.info("Running in single-zone environment - verifying cluster configuration")
+            
+            # Check if pod anti-affinity is configured in the cluster spec
+            get_cluster_cmd = [
+                "kubectl", "get", "cluster", cluster_info["name"],
+                "-n", cluster_info["namespace"],
+                "-o", "json"
+            ]
+            
+            result = subprocess.run(get_cluster_cmd, capture_output=True, text=True)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)